test/test_utils.c

```diff
@@ -98,11 +99,13 @@ void assert_memcmp_impl(const char *file, int line,
 }
 
 /* Stubs: see test_utils.h */
-struct mp_log *mp_null_log;
+struct mp_log *const mp_null_log;
 const char *mp_help_text;
 
 void mp_msg(struct mp_log *log, int lev, const char *format, ...) {};
 int mp_msg_find_level(const char *s) {return 0;};
 int mp_msg_level(struct mp_log *log) {return 0;};
 void mp_write_console_ansi(void) {};
 void mp_set_avdict(AVDictionary **dict, char **kv) {};
+struct mp_log *mp_log_new(void *talloc_ctx, struct mp_log *parent,
+                          const char *name) { return NULL; };
```

Comment:
Should be declared in `test_utils.h` too.

Reply:
Why? I don't understand why we link object selectively tests, but I followed this notation to fix linking error.

Why not use `common/msg.h`?

Reply:
I just found it easier at the time. Using trying to `common/msg.h` instead of `test_utils.h` leads to a bunch of warnings, undefined references, etc.

Reply:
But current way, doesn't bring warnings, I'm not linking `msg.c`, just header for declarations.

Reply:
Okay, I swore this was giving me the function prototype warning earlier but it seems to not be happening anymore.